src/cljs/rems/fields.cljs

```diff
@@ -97,7 +114,13 @@
       " "
       (if optional
         (text :t.form/optional)
-        (text :t.form/required))]
+        (text :t.form/required))
+      (when info-text
```

Comment:
@opqdonut what do you think?

Reply:
It probably mostly works, but what about when a user fills in the info text in form editor but then clears it? I think you might end up with a `""` info text in that case.

Reply:
hmm, true. maybe I should do an empty string check?

Reply:
yep! remember that info-text is a map here